Amplify/DevMenu/View/DeviceInfoDetailView.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+import UIKit
+
+/// Detail view containing device information
+@available(iOS 13.0.0, *)
+struct DeviceInfoDetailView: View {
+
+    private let screenTitle = "Device Information"
+    public static let deviceNameKey = "Device Name"
+    public static let systemNameKey = "System Name"
+    public static let systemVersionKey = "System Version"
+    public static let modelNameKey = "Model Name"
+    public static let localizedModelNameKey = "Localized Model Name"
```

Comment:
Some other things we could add:

- `isSimulator`
- `hasInternetConnection`
- `internetConnectionType` (wifi, lte, 3g, edge, etc)

Reply:
I added isSimulator field. For internet connection, I need to investigate more and will address in separate PR.

Reply:
sounds good!